entity-registry/src/test_plugins/mycompany-full-model/0.0.1/entity-registry.yaml

```diff
@@ -5,4 +5,9 @@ entities:
   - name: newThing
     keyAspect: testDataQualityRules
     aspects:
-      - testDataQualityRules
\ No newline at end of file
+      - testDataQualityRules
+plugins:
```

Comment:
is the plugins section required? this would be a backwards incompatible change with previous yaml spec

Reply:
It is not required, I can remove this. It was added as a placeholder before I wrote the other test. One additional thought here is that we need a test around merging this section of the file.